content/docs/communities/administration/_index.md

```diff
@@ -0,0 +1,166 @@
+Matrix does not prescribe an order in which to upgrade, so "room downgrades" or "room sidegrades" are possible using the same process.
+For example, rooms can have certain settings that are fixed and cannot be changed in hindsight.
+This includes for example enabling end-to-end encryption, or determining the room creator.
+Similarly, this process could be used in rooms up to version 12 to remove admins from a room or to adjust creators from room version 12 onwards.
+
+<!-- TODO: Fixing split rooms, see chapter below? -->
+
+On August 11th 2025 there is a [planned](@/blog/2025/07/2025-07-16-security-predisclosure.md) coordinated security release intoducing the new room version 12, with a security disclosure following on August 14th.
+Affected rooms should be upgraded within a reasonable time frame.
+
+<!-- TODO: in which cases is this relevant? as far as we know rooms that may contain malicious homeservers, i.e. primarily public rooms -->
+
+### How to upgrade a room
+
+{% notice_box() %}
+We recommend to read this chapter first in its entirety and make a plan before starting to execute any upgrades.
+Room upgrades are not a frictionless process and you want to get it right first try to cause as little friction to your community as possible.
+{% end %}
+
+**Prerequisites:**  
+Executing a room upgrade in its entirety, including establishing the link between the rooms, requires the executing user to have a sufficient amount of rights ("[power level](@/docs/communities/moderation/_index.md#power-levels)" to send a [`m.room.tombstone`](https://spec.matrix.org/latest/client-server-api/#server-behaviour-19) event) in the room that they want to upgrade.
+
+**Upgrading:**  
+Clients that [support upgrading rooms](#sending-room-upgrades) usually offer a button or similar in their UI when an upgrade is available.
+Sometimes the client UI might allow you to choose the target room version, but usually the default room version defined by the [homeserver](@/docs/matrix-concepts/elements-of-matrix/_index.md#homeserver) is used.
+
+If the option is missing unexpectedly, it is possible that the combination of your client's UI and your homeserver's configuration does not recommend an upgrade at this time.
+Read on to learn how to upgrade anyway.
+
+**Advanced upgrading:**  
+There are some reasons why your client might not be showing you the upgrading UI despite an upgrade being available, including:
+- Your homeserver implementation or admin doesn't recommend it, via the [homeserver's configuration](https://spec.matrix.org/latest/client-server-api/#mroom_versions-capability).
+- Your client only recommends upgrading from or between certain versions.
+
+In these cases you can use e.g. Element Web's `/upgraderoom <version>` chat command after [enabling the developer tools](https://docs.element.io/latest/element-support/frequently-asked-questions/?h=devtools#matrix-general).
```

Comment:
```suggestion
In these cases you can use e.g. Element Web's `/upgraderoom <version>` chat command.
```

Reply:
As [mentioned](https://github.com/matrix-org/matrix.org/pull/2861/files#r2266483887) I'm wary of this.